.changeset/nervous-dogs-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-proxy-backend': minor
```

Comment:
While on 0.x we're marking non-breaking changes as `patch` level, which I think this should be